sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 
-## 4.5.0-beta.1 (Unreleased)
-
+## 4.5.0 (2020-09-16)
+* Increased robustness of the Rntbd stack in case of intermittent connectivity issues
+* Improved latency in case of intermittent connectivity issues to individual backend replicas for multi-region accounts avoiding initiation of unnecessary regional fail-overs 
```

Comment:
```suggestion
* Improved latency in case of intermittent connectivity issues to individual backend replicas for multi-region accounts avoiding initiation of unnecessary regional fail-overs 
```